### ISSUE TITLE
dynamicbungee list command doesn't work

### ISSUE THREAD

**Issue by lunaseraphica** [mineglade/dynamic-bungee#3]
Ok so the dynamicbungee list command does not want to work as seen in the video idk why, can you fix that please?

**Comment by mia-riezebos** [comment]
Hi, so sorry for the exTREMELy delayed response, for some reason my github didn't notify me of this issue. I will investigate and get back to you soon.
I'm in the middle of a move right now, though so it might take some time.

**Comment by mia-riezebos** [comment]
right, it's been a while since i'd taken a look at this project. I mentioned in the release notes for the beta 0.3-preview update, that the list command has been turned off for further testing. 
I remember I was using a library to format the list into a table, however this didn't translate well into chatcomponents, so I was exploring other options.
I think it's stable enough to fork & turn it back on (pretty sure it's just commented out in code), so you could build your own jar for that purpose. I can release a quick fix with a config toggle for the list command this week, for those unable to build it themselves.

pre-requisites for local builds:
  - maven
  - https://github.com/derkades/derkutils (version: master)
    - maven install derkutils to use in dynbun project :)
